### PR TITLE
Issue 6 services pages

### PR DESCRIPTION
**Pull request by HuxleyBerry**
## Change Summary
- Made a reusable component for section headers
- Built the core services page and styled it to look like the figma
  - Used some more reusable components here too
- Made it so that the margins and some of the icon sizes are reduced on mobile

Things that haven't been done:
- The image at the top needs to be cropped to match the figma (I wasn't sure if the image was just a placeholder so I didn't bother)
- The icons need to be updated to the ones Jules made

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
* I've occasionally noticed that the --starick-olive background colour doesn't show up and instead displays as white. I don't know what is going on there

### CODE REVIEW

File: .gitignore
Comment (cjnbennett) [inline .gitignore:1]: CFC prefers to commit .vscode/*

**Review by cjnbennett — CHANGES_REQUESTED** [review]
Looks good, but remove .vscode from gitignore. Also, do you have an example of where --starick-olive doesn't work?

File: .gitignore
Comment (HuxleyBerry) [reply .gitignore:1]: The refuges and support in police stations section headers sometimes show up for me with white backgrounds rather than olive.

**Comment by cjnbennett** [comment]
I believe the --starick-olive not working sometimes is because tailwind only loads classes that it finds being used during a static check. It checks some scope including the section-title.tsx file and sees that --starick-olive doesn't occur anywhere so doesn't bother to add a class for it. The way to get around this is to use a [safelist](https://tailwindcss.com/docs/content-configuration#safelisting-classes). Just pushed a commit doing this.

**Review by cjnbennett — APPROVED** [review]
Looks good now, feel free to merge

**Pull request merged**